ruby.md

```diff
@@ -5,41 +5,56 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. In what ways are JavaScript and Ruby similar? In what ways are they different?
 
-  Your answer:
+  Your answer: They are both object oriented. 
 
-  Researched answer:
+  Researched answer: Both languagues are object-oriented, but in Ruby, everything is an object. JS is primarily a front-end language, while Ruby is primarily a back-end one.
 
 
 
 2. What is a hash?
 
-  Your answer:
+  Your answer: A collection of key-value pairs. 
 
-  Researched answer:
+  Researched answer: Hashes are a way of looking up data by assigning value to a key. Whenever that value needs to be referenced, the key is used to look it up. 
 
 
 
 3. What is the testing framework used in Ruby? Describe the process of setting up the testing environment.
 
-  Your answer:
+  Your answer: Behavior Driven Testing
 
```

Comment:
BDT is more of a programming philosophy. Rspec is the testing framework we used in Ruby.

Reply:
I thought so at first but I guess I just got it backwards. whoops!